opencti-platform/opencti-front/src/private/components/analyses/external_references/ExternalReference.tsx

```diff
@@ -39,13 +40,18 @@ const ExternalReferenceComponent: FunctionComponent<
 ExternalReferenceComponentProps
 > = ({ externalReference, connectorsImport }) => {
   const classes = useStyles();
+  const { isFeatureEnable } = useHelper();
+  const FABReplacement = isFeatureEnable('FAB_REPLACEMENT');
   return (
     <div className={classes.container}>
       <ExternalReferenceHeader
         externalReference={externalReference}
         PopoverComponent={
           <ExternalReferencePopover id={''} handleRemove={undefined} />
         }
+        EditComponent={<Security needs={[KNOWLEDGE_KNUPDATE]}>
+          <ExternalReferenceEdition externalReferenceId={externalReference.id} />
+        </Security>}
```

Comment:
For consistency, I advise formatting like this::

```suggestion
        EditComponent={(
          <Security needs={[KNOWLEDGE_KNUPDATE]}>
            <ExternalReferenceEdition externalReferenceId={externalReference.id} />
          </Security>
        )}
```